migrations/v2.js

```diff
@@ -0,0 +1,37 @@
+import { describe , whereContent, whereFromPlugin, whereToPlugin, mutateContent, checkContent, updatePlugin } from 'adapt-migrations';
+
+function hasKey(object, key) {
+  if (!object) return false;
+  return Object.hasOwn(object, key);
+}
+
+function setObjectPathValue(object, path, value, force = false) {
+  if (!object) return;
+  const paths = path.split('.');
+  const key = paths.pop();
+  const target = paths.reduce((o, p) => {
+    if (!hasKey(o, p)) o[p] = {};
+    return o?.[p];
+  }, object);
+  if (!force && hasKey(target, key)) return;
+  target[key] = value;
+}
+
+describe('adapt-contrib-text - v2.0.0 to v2.0.1', async () => {
+  whereFromPlugin('adapt-contrib-text - from v2.0.0 to v2.0.1', { name: 'adapt-contrib-text', version: '<2.0.1'});
+  let course;
+  whereContent('adapt-contrib-text - where missing _globals ariaRegion', async content => {
+    course = content.find(({ _type }) => _type === 'course');
+    return !hasKey(course?._globals?._components?._text, 'ariaRegion');
```

Comment:
As mentioned we'll be changing to _.has (I'll not raise a comment for every instance),

```suggestion
    return !_.has(course, '_globals._components._text.ariaRegion');
```